### PR TITLE
Add lrm-valhalla tutorial

### PR DESCRIPTION
**Pull request by rmglennon**

### CODE REVIEW

File: lrm-valhalla-walkthrough.md [hunk 1]
```diff
@@ -0,0 +1,293 @@
...
+      L.latLng(41.5893,-90.5715)
+    ],
+    router: L.Routing.valhalla('valhalla-xxxxxx', 'auto'),
+    formatter: new L.Routing.Valhalla.Formatter(),
+    summaryTemplate:'<div class="start">{name}</div><div class="info {transitmode}">{distance}, {time}</div>',
+    routeWhileDragging: false
+  }).addTo(map);
+ ```
+
+When you refresh the map, you should see the map, route line, and updated icons and summary text in the narration box. You can ‘auto’ to ‘pedestrian’ or ‘bicycle’ if you want to build a route for walking or biking, although this particular route would exceed Valhalla's limits for other modes of transport.
```
Comment (kevinkreiser) [inline lrm-valhalla-walkthrough.md]: might be missing a word here. maybe: "you can change 'auto' to..."

Comment (mjcunningham) [inline lrm-valhalla-walkthrough.md]: Agreed, but other than that this is great.

Comment (kevinkreiser) [inline lrm-valhalla-walkthrough.md]: indeed, so little javascript, so much joy ;)

**Comment by rmglennon** [comment]
Thanks @kevinkreiser and @mjcunningham. I'll fix the typo.

I think this should be ready for testing. We'll have to see whether performance is OK (and it's not too resource-intensive for many people to be doing it) with a route this long.

**Pull request merged**